src/components/WalletImageSlider.tsx

```diff
@@ -16,7 +17,7 @@ export default function WalletImageSlider({
     if (sliderValue >= index) return true;
   };
 
-  const handleEventChange = (event) => {
+  const handleEventChange = (event: any) => {
```

Comment:
When using typescript, try not to use any

Reply:
addressed.